app/services/visit_request/create.rb

```diff
@@ -0,0 +1,26 @@
+class VisitRequest
+  class Create < ApplicationService
+    def initialize(user, event)
+      @user  = user
+      @event = event
+    end
+
+    def call
+      return visit_request.approved! if user.verified? && event.has_free_slot_for?(user)
+
+      visit_request.pending! unless user.verified?
+      visit_request.pending! unless event.has_free_slot_for?(user)
```

Comment:
if these two conditions are false - is it means that 2 updates will be executed ?